### PR TITLE
Document JSONAsString

### PR DESCRIPTION
**Pull request by filimonov**
I hereby agree to the terms of the CLA available at: https://yandex.ru/legal/cla/?lang=en

Changelog category (leave one):
- Documentation (changelog entry is not required)

Changelog entry (a user-readable short description of the changes that goes to CHANGELOG.md):
Document JSONAsString

Detailed description / Documentation draft:

### CODE REVIEW

File: docs/en/interfaces/formats.md [hunk 1]
```diff
@@ -25,7 +25,8 @@ The supported formats are:
 | [Vertical](#vertical)                                                                   | ✗     | ✔      |
 | [VerticalRaw](#verticalraw)                                                             | ✗     | ✔      |
 | [JSON](#json)                                                                           | ✗     | ✔      |
-| [JSONString](#jsonstring)                                                               | ✗     | ✔      |
+| [JSONAsString](#json)                                                                   | ✗     | ✗      |
```
Comment (alexey-milovidov) [inline docs/en/interfaces/formats.md:28]: Looks strange.

File: docs/en/interfaces/formats.md [hunk 2]
```diff
@@ -507,6 +508,34 @@ Example:
 }
 ```
 
+## JSONAsString {#jsonasstring}
```
Comment (alexey-milovidov) [inline docs/en/interfaces/formats.md:511]: LGTM

File: docs/en/interfaces/formats.md [hunk 3]
```diff
@@ -25,8 +25,8 @@ The supported formats are:
 | [Vertical](#vertical)                                                                   | ✗     | ✔      |
 | [VerticalRaw](#verticalraw)                                                             | ✗     | ✔      |
 | [JSON](#json)                                                                           | ✗     | ✔      |
-| [JSONAsString](#json)                                                                   | ✗     | ✗      |
-| [JSONString](#jsonasstring)                                                             | ✔     | ✔      |
+| [JSONAsString](#jsonasstring)                                                           | ✔     | ✗      |
+| [JSONString](#jsonstring)                                                               | ✗     | ✔      |
```
Comment (alexey-milovidov) [inline docs/en/interfaces/formats.md:29]: JSONStrings maybe?

Comment (filimonov) [reply docs/en/interfaces/formats.md:29]: It was there here before. I would need to check what's this (too many JSON options actually, very confusing)

**Pull request merged**